src/components/MemberContainer.js

```diff
@@ -0,0 +1,79 @@
+import React, {useState} from 'react'
+import themeGet from '@styled-system/theme-get'
+import {Flex, Box, theme} from '@primer/components'
+import MemberQuestions from './MemberQuestions'
+import AvatarShape from './AvatarShape'
+import {ReactComponent as DotsSVG} from '../svg/dots.svg'
+
+const Dots = ({shape, color}) => {
+  const css = {
+    position: 'absolute',
+    height: '92px',
+    width: '168px',
+    fill: color
+  }
+  return <DotsSVG shape={shape} style={Object.assign(css, shapeStyles[shape])} />
```

Comment:
Interesting. This still works even though `shapeStyles` is defined at the end of the file? I guess it's defined before the render function runs 🤷‍♂